src/Sylius/Bundle/CurrencyBundle/Resources/config/validation/ExchangeRate.xml

```diff
@@ -29,6 +29,11 @@
                 <option name="message">sylius.exchange_rate.ratio.greater_than</option>
                 <option name="groups">sylius</option>
             </constraint>
+            <constraint name="LessThan">
+                <option name="value">100000</option>
```

Comment:
Is it the maximum possible float value?

Reply:
Right now we have the mapping: `<field name="ratio" column="ratio" type="decimal" precision="10" scale="5" />` which makes maximum possible value `99999.99999`